### PR TITLE
disallow other browsers

### PR DESCRIPTION
**Pull request by Geczy**
only allow chrome

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -33,14 +38,36 @@ export default function App() {
...
+          <MenuBar />
+          <Main>
+            {roomState === 'disconnected' ? <LocalVideoPreview /> : <Room />}
+            <Controls />
+          </Main>
+          <ReconnectingNotification />
+        </Container>
+      );
+
+    default:
```
Comment (RusseII) [inline src/App.tsx:57]: What about other browsers that are supported, but are not chrome or firefox? For example Brave, and a few other niche browsers (and edge, chromium version)

I know i've seen people suggest testing for browser capability, over browser type

File: src/App.tsx [hunk 2]
```diff
@@ -23,6 +25,9 @@ const Main = styled('main')({
   overflow: 'hidden',
 });
 
+const { detect } = require('detect-browser');
```
Comment (RusseII) [inline src/App.tsx:28]: I've used this library before - https://github.com/DamonOehlman/detect-browser/issues/110 

The library does not seem to stay up to date with the most current user agents and sometimes can't detect certain browser types

I'd much rather check of the browser supports the needed APIs for live video

File: src/App.tsx [hunk 3]
```diff
@@ -33,14 +38,36 @@ export default function App() {
...
+            {roomState === 'disconnected' ? <LocalVideoPreview /> : <Room />}
+            <Controls />
+          </Main>
+          <ReconnectingNotification />
+        </Container>
+      );
+
+    default:
+      return (
+        <Container style={{ height }}>
```
Comment (RusseII) [inline src/App.tsx:59]: Great looking chrome screen!

File: src/App.tsx (hunk 1 above)
Comment (Geczy) [reply src/App.tsx:57]: In this library, specifying `chrome` means `chromium` engine

File: src/App.tsx (hunk 2 above)
Comment (Geczy) [reply src/App.tsx:28]: Dang

**Comment by Geczy** [comment]
I recommend banning any browser that isn't Chrome or Firefox

**Comment by RusseII** [comment]
Thanks @Geczy. Merged your work into #18

**Pull request closed without merging**